src/socketio.quiver.js

```diff
@@ -40,6 +43,19 @@ function QuiverSocialProvider(dispatchEvent) {
   this.dispatchEvent = dispatchEvent;
   this.storage = freedom['core.storage']();
 
+  /** @private @const {!FreedomCrypto} */
+  this.crypto_ = freedom['core.crypto']();
+
+  /** @private @const {!FreedomPgp} */
+  this.pgp_ = freedom['pgp-e2e']();
+
+  /** @type {!Promise<string>} */
+  this.pubKey_ = this.pgp_.setup('', '<quiver>').then(function() {
```

Comment:
Can we call this getPubKey_ or loadPubKey_ rather than just pubKey_?  That will help clarify that it's actually a Promise